### PR TITLE
10 Change prompt language to english

### PR DESCRIPTION
**Pull request by SamuelPelletierEvraire**
see issue  #10

### CODE REVIEW

**Comment by snakedye** [comment]
Idealement la prompt devrait être en anglais vu que le LLM est plus train dessus.

File: prompt.txt [hunk 1]
```diff
@@ -0,0 +1,62 @@
+Tu es un inspecteur d'étiquette d'engrais qui travaille à l'agence d'inspection alimentaire canadienne. Mise en contexte : un texte extrait d'une image t'est fournie.
+Ton but est de classer l'entièreté des informations présente sans les modifier grâce à la liste de clé suivante.
+Tu dois absolument utiliser les clés dans la liste suivante.
+Ta réponse est exacte et non généré par LLM.
+Ta réponse est sous le format JSON.
+Ta réponse contient tout le texte présent dans le texte fourni.
+Avant de rendre ta réponse, vérifie celle-ci afin qu'elle concorde avec tous les points précédents.
+Voici la liste de clés :
+"compagny_name";
```
Comment (snakedye) [inline prompt.txt:9]: company not compagny

File: prompt.txt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+"fertiliser_cautions_EN";
+"fertiliser_recommendation_FR";
+"fertiliser_recommendation_EN";
+"fertiliser_first_aid_FR";
+"fertiliser_first_aid_EN";
+"fertiliser_warranty_FR";
+"fertiliser_warranty_EN";
+"fertiliser_danger_FR";
+"fertiliser_danger_EN";
+"fertiliser_guaranteed_analysis", sous-section "nutrient_in_guaranteed_analysis",  sous-section "percentage_in_guaranteed_analysis";
```
Comment (snakedye) [inline prompt.txt:39]: this results in two fields. what we actually want is the nutrient as a key and the percentage as the value. something like this:
```json
"fertilizer_guarantee_analysis": {
  "Potasium": "90%",
  "Zinc": "5%",
}
```
or a table
```json
"fertilizer_guarantee_analysis": [
  ["Potasium", "90%"],
  ["Zinc", "5%"],
]
```

Comment (snakedye) [reply prompt.txt:39]: Table is better

File: prompt.txt [hunk 3]
```diff
@@ -0,0 +1,62 @@
+Tu es un inspecteur d'étiquette d'engrais qui travaille à l'agence d'inspection alimentaire canadienne. Mise en contexte : un texte extrait d'une image t'est fournie.
```
Comment (snakedye) [inline prompt.txt:1]:
```suggestion
You have an unorderer text extracted from a fertilizer label image.
```

**Review by snakedye — COMMENTED** [review]
Obviously this should be in english. There also seem to be quite a lot of conflicts now so rebase it on main.

**Comment by Endlessflow** [comment]
This PR has been resolved through a different, more elegant solution provided by PR #67. It appears we can safely close this PR.

**Pull request closed without merging**